### PR TITLE
chore(v1): Remove IE Support

### PR DESCRIPTION
**Pull request by adamhock**
- Checks for if the browser supports CSS variables have been removed
- Non-animated tabs have been removed

### CODE REVIEW

File: packages/iTwinUI-react/src/core/Badge/Badge.tsx [hunk 1]
```diff
@@ -70,16 +69,10 @@ export const Badge = (props: BadgeProps) => {
...
-      `--iui-badge-background-color: ${backgroundColor}`,
-    )
-      ? {
-          '--iui-badge-background-color': getBadgeColorValue(backgroundColor),
-          ...style,
-        }
-      : { backgroundColor: getBadgeColorValue(backgroundColor), ...style };
+  const _style = backgroundColor && {
+    '--iui-badge-background-color': getBadgeColorValue(backgroundColor),
+    ...style,
```
Comment (mayank99) [inline packages/iTwinUI-react/src/core/Badge/Badge.tsx:74]: The user-passed `style` will not be applied if `backgroundColor` is not passed

File: packages/iTwinUI-react/src/core/ColorPicker/ColorSwatch.tsx [hunk 2]
```diff
@@ -39,12 +39,7 @@ export const ColorSwatch = React.forwardRef<HTMLDivElement, ColorSwatchProps>(
     );
 
     const _style = React.useMemo(
-      () =>
-        getWindow()?.CSS?.supports?.(
-          `--iui-color-swatch-background: ${colorString}`,
-        )
-          ? { '--iui-color-swatch-background': colorString, ...style }
-          : { backgroundColor: colorString, ...style },
+      () => ({ '--iui-color-swatch-background': colorString, ...style }),
```
Comment (mayank99) [inline packages/iTwinUI-react/src/core/ColorPicker/ColorSwatch.tsx:42]: There is no need to memo this anymore

File: packages/iTwinUI-react/src/core/Tree/TreeNode.tsx [hunk 3]
```diff
@@ -130,10 +125,7 @@ export const TreeNode = (props: TreeNodeProps) => {
   const nodeRef = React.useRef<HTMLLIElement>(null);
 
   const styleDepth = React.useMemo(
-    () =>
-      getWindow()?.CSS?.supports?.(`--level: ${nodeDepth}`)
-        ? { '--level': nodeDepth }
-        : { marginLeft: nodeDepth ? nodeDepth * 28 : 0 },
+    () => ({ '--level': nodeDepth } as React.CSSProperties),
```
Comment (mayank99) [inline packages/iTwinUI-react/src/core/Tree/TreeNode.tsx:128]: no need to memo this. i would say move it to inline

Comment (adamhock) [reply packages/iTwinUI-react/src/core/Tree/TreeNode.tsx]: Fixed

File: packages/iTwinUI-react/src/core/ColorPicker/ColorSwatch.tsx (hunk 2 above)
Comment (adamhock) [reply packages/iTwinUI-react/src/core/ColorPicker/ColorSwatch.tsx]: Fixed

**Review by mayank99 — APPROVED** [review]

**Review by elephantcatdog — COMMENTED** [review]
Everything you have looks good to me. I found two places that might need changing also:
1. In ComboBoxMenu.tsx there's a check for `supports` 
image
Does this need to be removed/simplified?
2. In Table.tsx there are 3 places where polyfills are mentioned in the comments on lines 148, 153, & 237.
image
Should we remove these IE references?

**Comment by adamhock** [comment]
> Everything you have looks good to me. I found two places that might need changing also:
> 
> 1. In ComboBoxMenu.tsx there's a check for `supports`
>    image
>    Does this need to be removed/simplified?
> 2. In Table.tsx there are 3 places where polyfills are mentioned in the comments on lines 148, 153, & 237.
>    image
>    Should we remove these IE references?

`overflow-x: overlay` does not work for Firefox, so we have to keep this check

The comments for IE are removed

**Review by elephantcatdog — APPROVED** [review]
Looks good!

**Pull request merged**